content/user/apps/Session-metadata.md

```diff
@@ -0,0 +1,68 @@
+<!--
+title: "Session Metadata"
+description: "Set up session metadata to track applications' vulnerabilities"
+tags: "user UI applications session metadata vulnerabilities"
+-->
+
+Pinpoint the source of vulnerabilities in your application with session metadata reported by your Contrast agent. As soon as you add the necessary configuration property to your agent file, the agent reports the information along with the rest of your standard vulnerability data to the Contrast UI. 
+
+## Agent Configuration 
+
+To send session metadata for your application to the Contrast UI, you must add the configuration settings to your agent configuration file. The build properties that the agent can report are **branch name**, **build number**, **commit hash**, **committer**, **Git tag**, **repository**, **test run** and **version**. You may include all or some of these properties, as desired. 
+
+You can supply these settings as system properties, environment settings or properties in a YAML configuration file. See the following configuration examples for each application language.
+
+### Examples
+ 
+If you use system properties for [Java](installation-javaconfig.html), include an additional entry in the line where you add your `javaagent` flag. In this case, you will set the property `contrast.application.session_metadata` to a set of key-value pairs that identify your test run. 
+
+```
+-Dcontrast.application.session_metadata="branchName=feature/some-new-thing,committer=Jane,repository=Contrast-Java"
+```
+ 
+As a [.NET](installation-netconfig.html) developer using *app.config* or *web.config*, you can add an entry to your configuration to specify this property. 
+
+```
+<?xml version="1.0"?>
+<configuration>
+  <connectionStrings />
+  <appSettings>
+    <add key="contrast.application.session_metadata" value="branchName=feature/some-new-thing,committer=Jane,repository=Contrast-DotNet" />
+```
+ 
+If you use YAML configuration, like most [Node](installation-nodeconfig.html) and [Ruby](installation-rubyconfig.html) agents, you can add an additional entry to your *contrast_security.yaml* file. 
+
+```
+application:
+ session_metadata:branchName=feature/some-new-thing,committer=Jane,repository=Contrast-Ruby
+```
+ 
```

Comment:
`session_metadata: branchName=feature/some-new-thing,committer=Jane,repository=Contrast-Ruby`

(add space between colon and `branchName`)